pkg/ovndbcluster/volumes.go

```diff
@@ -4,7 +4,8 @@ import corev1 "k8s.io/api/core/v1"
 
 // GetDBClusterVolumes -
 // TODO: merge to GetVolumes when other controllers also switched to current config
-//       mechanism.
+//
+//	mechanism.
```

Comment:
the extra empty line probably unnecessary if the empty spaces are removed before 'mechanism.'

Reply:
These changes were by the pre-commit hook itself. Changed.

Reply:
Yeah, it probably misunderstood the extra spaces before the first word and moved it to a separate paragraph. Thanks for fixing.